modules/liberty/src/main/java/org/testcontainers/containers/LibertyServerContainer.java

```diff
@@ -0,0 +1,145 @@
+package org.testcontainers.containers;
+
+import lombok.NonNull;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+import org.testcontainers.applicationserver.ApplicationServerContainer;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Represents an Open Liberty or WebSphere Liberty container
+ */
+public class LibertyServerContainer extends ApplicationServerContainer {
+
+    public static final String NAME = "Liberty";
+
+    // About the image
+    public static final String IMAGE = "open-liberty";
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse(IMAGE);
+
+    // Container defaults
+    public static final int DEFAULT_HTTP_PORT = 9080;
+
+    public static final int DEFAULT_HTTPS_PORT = 9443;
```

Comment:
private or package protected modifier are ok due to are just used by the container implementation